addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -62,6 +62,8 @@ import {
     getSelectedNodes,
     isVisibleStr,
     pxToFloat,
+    descendants,
```

Comment:
@dmo-odoo Here I squashed the corrections into one commit. If you validate I'll redistribute all this into the right commits.

Reply:
Alright for the corrections in the commit. Some comments are still unresolved though. Not sure if it was intended or not... ^^'

Reply:
It wasn't, I'm looking now.